libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/atomic/atoms/ElementLogoAtom.kt

```diff
@@ -53,85 +53,82 @@ import io.element.android.libraries.theme.ElementTheme
+    val shadowColor = if (darkTheme) size.shadowColorDark else size.shadowColorLight
+    val backgroundColor = if (darkTheme) Color.White.copy(alpha = 0.2f) else Color.White.copy(alpha = 0.4f)
+    val borderColor = if (darkTheme) Color.White.copy(alpha = 0.8f) else Color.White.copy(alpha = 0.4f)
     Box(
         modifier = modifier
-            .size(outerSize)
-            .border(borderWidth, borderColor, RoundedCornerShape(cornerRadius)),
+            .size(size.outerSize)
+            .border(size.borderWidth, borderColor, RoundedCornerShape(size.cornerRadius)),
         contentAlignment = Alignment.Center,
     ) {
         Box(
             Modifier
-                .size(outerSize)
+                .size(size.outerSize)
                 .shapeShadow(
                     color = shadowColor,
-                    cornerRadius = cornerRadius,
-                    blurRadius = 32.dp,
+                    cornerRadius = size.cornerRadius,
+                    blurRadius = size.shadowRadius,
                     offsetY = 8.dp,
                 )
         )
         Box(
             Modifier
-                .clip(RoundedCornerShape(cornerRadius))
-                .size(outerSize)
+                .clip(RoundedCornerShape(size.cornerRadius))
+                .size(size.outerSize)
                 .background(backgroundColor)
                 .blur(blur)
         )
         Image(
-            modifier = Modifier.size(logoSize),
+            modifier = Modifier.size(size.logoSize),
             painter = painterResource(id = R.drawable.element_logo),
             contentDescription = null
         )
     }
 }
 
-enum class ElementLogoAtomSize {
-    Medium,
-    Large
-}
+sealed class ElementLogoAtomSize(
+    val outerSize: Dp,
+    val logoSize: Dp,
+    val cornerRadius: Dp,
+    val borderWidth: Dp,
+    val shadowColorDark: Color,
+    val shadowColorLight: Color,
+    val shadowRadius: Dp,
+) {
+    object Medium : ElementLogoAtomSize(
+        outerSize = 120.dp,
+        logoSize = 83.5.dp,
+        cornerRadius = 33.dp,
+        borderWidth = 0.38.dp,
+        shadowColorDark = Color.Black.copy(alpha = 0.4f),
+        shadowColorLight = Color(0x401B1D22),
+        shadowRadius = 32.dp,
+    )
 
-@Composable
-@DayNightPreviews
-internal fun ElementLogoAtomPreview() {
-    ElementPreview {
-        Box(
-            Modifier
-                .size(180.dp)
-                .background(ElementTheme.colors.bgSubtlePrimary),
-            contentAlignment = Alignment.Center
-        ) {
-            ElementLogoAtom(ElementLogoAtomSize.Large)
-        }
-    }
+    object Large : ElementLogoAtomSize(
+        outerSize = 158.dp,
+        logoSize = 110.dp,
+        cornerRadius = 44.dp,
+        borderWidth = 1.dp,
+        shadowColorDark = Color.Black.copy(alpha = 0.4f),
+        shadowColorLight = Color(0x401B1D22),
+        shadowRadius = 32.dp,
+    )
+
+    object Huge : ElementLogoAtomSize(
```

Comment:
I actually created the `Large` size for this screen, but I forgot to replace it. You can just remove `Huge` and customize the `Large` one.

Reply:
Ah OK, thanks, I forgot to check for usage :)